pkg/openthread/Makefile.dep

```diff
@@ -4,4 +4,10 @@ USEMODULE += openthread_contrib_netdev
 USEMODULE += l2util
 USEMODULE += xtimer
 USEMODULE += event
+USEMODULE += netdev_default
```

Comment:
That shouldn't be up to the pkg to choose.
Just remove that line, it's usually set by application (and squash again)